packages/order-experience-api/src/lib/validation.ts

```diff
@@ -41,3 +41,15 @@ export const customerSchema = yup.object().shape({
 export const validateCustomer = (p: OrderCustomer): Promise<boolean> => {
   return customerSchema.isValid(p)
 }
+
+export const paymentFiltersSchema = yup.array().of(
+  yup.object().shape({
+    filterId: yup.string(),
+    createdAt: yup.string(),
+    namespace: yup.string().required(),
+    referenceId: yup.string().required(),
+    referenceType: yup.string().required(),
```

Comment:
Can there be some kind of value validation,

`string().matches(/(order|merchant)/).required();`

https://github.com/jquense/yup#stringmatchesregex-regex-message-string--function-schema

Reply:
Yes sure, I'll add the specific value matching if possible.